kivy/core/clipboard/__init__.py

```diff
@@ -127,13 +131,16 @@ def _paste(self):
         ('winctypes', 'clipboard_winctypes', 'ClipboardWindows'))
 elif platform == 'linux':
     _clipboards.append(
-        ('xsel', 'clipboard_xsel', 'ClipboardXsel'))
-    _clipboards.append(
         ('dbusklipper', 'clipboard_dbusklipper', 'ClipboardDbusKlipper'))
+    _clipboards.append(
+        ('gtk3', 'clipboard_gtk3', 'ClipboardGtk3'))
+    _clipboards.append(
+        ('xsel', 'clipboard_xsel', 'ClipboardXsel'))
 
 if USE_SDL2:
-    _clipboards.append(
-        ('sdl2', 'clipboard_sdl2', 'ClipboardSDL2'))
+    if platform != 'linux':
```

Comment:
Make sure to open a bug report on sdl2 upstream, so that we can track when, if it is fixed.